### PR TITLE
Handling api token in more secure way

### LINKED ISSUE #240: Token for API
[A token is used for the API. ](http://oep-data-interface.readthedocs.io/en/latest/api/how_to.html#authenticate)

How do we handle/store the token in a secure way? We definitely should not hard code it in our python scripts and push it to GitHub. 
A small search gives us this options. Feel free to add other or comment:

- [**stups-tokens**](https://pypi.python.org/pypi/stups-tokens)
- store it in a **config file**, similar to [oemof.db](https://github.com/oemof/oemof.db)
- [**keyring**](https://pypi.python.org/pypi/keyring)

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Closes #240 

The user has the option of storing the token in an environment variable or in the config file directly.

### CODE REVIEW

File: api/tutorials/token_config.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from subprocess import run
+# untrack the change to this file (prevent pushing token to github)
+run(['git', 'update-index', '--assume-unchanged', __file__])
```
Comment (Bachibouzouk) [inline api/tutorials/token_config.py]: This line will prevent the changes to this file to be tracked by git upon the first time a user does `from token_config import get_oep_token()`.

As experienced user will prefer the environment variable option, I assumed that the chances a user commit and push their change after copy-pasting their token in the file to be pretty slim.

**Comment by christian-rli** [comment]
Nice trick! I tried it out and it checks out.

**Comment by Bachibouzouk** [comment]
Learned that these tutorial belong to the [example repo](https://github.com/OpenEnergyPlatform/examples), so I moved the commits with `cherry-pick` to https://github.com/OpenEnergyPlatform/examples/pull/61

**Comment by Bachibouzouk** [comment]
Relocated to https://github.com/OpenEnergyPlatform/examples/pull/61

**Pull request closed without merging**